custom_model_runner/datarobot_drum/drum/language_predictors/base_language_predictor.py

```diff
@@ -127,6 +132,24 @@ def _init_mlops(self):
         if not mlops_loaded:
             raise Exception("MLOps module was not imported: {}".format(mlops_import_error))
 
+        if to_bool(self._params.get("allow_dr_api_access")):
+            try:
+                self._deployment = dr.Deployment.get(deployment_id)
+                drift_settings = self._deployment.get_drift_tracking_settings()
+                self._feature_drift_enabled = drift_settings.get("feature_drift", True).get(
+                    "enabled", True
+                )
+                self._target_drift_enabled = drift_settings.get("target_drift", True).get(
+                    "enabled", True
+                )
+                self._predictions_data_collections_enabled = (
+                    self._deployment.get_predictions_data_collection_settings().get("enabled", True)
+                )
+            except Exception as e:
+                logger.warning(f"Failed to get deployment / settings: {e}")
+                traceback.format_exc()
```

Comment:
In the interest in keeping all the info together, why not just
```suggestion
                logger.warning(f"Failed to get deployment / settings: {e}", exc_info=True)
```